Configure

```diff
@@ -282,7 +282,7 @@ foreach ( reverse sort( 'aes', 'aria', 'bf', 'camellia', 'cast', 'des', 'dh',
 my %version = ();
 
 collect_information(
-    collect_from_file(catfile($srcdir,'VERSION')),
+    collect_from_file(catfile($srcdir,'VERSION.txt')),
```

Comment:
If you really must, I would call it `.dat` rather than `.txt`

Reply:
Reasoning: it contains version data, rather than free form text

Reply:
Also, there are two more spots to touch:

https://github.com/openssl/openssl/blob/d9c2fd51e2e278bc3f7793a104ff7b4879f6d63a/dev/release-aux/release-version-fn.sh#L39

https://github.com/openssl/openssl/blob/d9c2fd51e2e278bc3f7793a104ff7b4879f6d63a/dev/release-aux/release-version-fn.sh#L105

Reply:
Yeah, that's better.

Reply:
> Also, there are two more spots to touch:

Ah, thanks. And I found and updated a further reference to `VERSION` in `dev/release.s`.